README.markdown

```diff
@@ -41,7 +41,6 @@ Descriptive and consistent naming makes software easier to read and understand.
 - use camel case (not snake case)
 - use uppercase for types and protocols, lowercase for everything else
 - name functions and parameters based on role, not on type
-- strive to name functions in a way that reads like a sentence
```

Comment:
Are we not trying to do this? I at least think it's great when a method follow this.

Reply:
We voted to get rid of this, after discussion of the really long and verbose type and function names, differing to the more general guidance on naming instead.